process/interface.go

```diff
@@ -237,6 +237,13 @@ type TransactionLogProcessor interface {
 	IsInterfaceNil() bool
 }
 
+// TransactionLogProcessorDatabase is interface the  for saving logs also in RAM
+type TransactionLogProcessorDatabase interface {
+	GetLogFromRAM(txHash []byte) (data.LogHandler, bool)
+	SaveLogsAlsoInRAM()
+	RemoveLogsFromRAM()
+}
```

Comment:
add IsInterfaceNil . change RAM to Cache, saveLogsToCache

Reply:
done.